pkg/provider/util/project_start_script.go

```diff
@@ -6,5 +6,46 @@ package util
 import "fmt"
 
 func GetProjectStartScript(daytonaDownloadUrl string, apiKey string) string {
-	return fmt.Sprintf(`curl -sfL -H "Authorization: Bearer %s" %s | sudo -E bash && daytona agent`, apiKey, daytonaDownloadUrl)
+	return fmt.Sprintf(`
+  # Check for missing dependencies
```

Comment:
I suggest first checking for `sudo` and installing it if it's missing.

The other commands should then use sudo (e.g. `sudo apt-get update`...).

`sudo` is required most of the time to run these commands.

Reply:
done